comarev/comarev-dashboard
# Chore Add e2e action to the CI

Closes: #82 

#### What?

- Add an action to run e2e tests

#### Why?

- To ensure we have all the e2e passing without errors for every new PR

#### How to test it?

- The GitHub action should trigger the action to run when the PR is opened but you can run `yarn run test:e2e`
